src/app/web-app.component.ts

```diff
@@ -19,13 +19,17 @@ import { Logger } from './core/logger/logger.service';
 import { I18nService } from './core/i18n/i18n.service';
 import { ThemeStorageService } from './shared/theme-picker/theme-storage.service';
 import { AlertService } from './core/alert/alert.service';
+import { KeyboardShortcutsService } from './help/keyboard-shortcuts.service';
 
 /** Custom Models */
 import { Alert } from './core/alert/alert.model';
 
 /** Initialize Logger */
 const log = new Logger('MifosX');
 
+type Unlisten = () => void;
```

Comment:
Please explain the use of this variable in a proper comment.

Reply:
Done.